Reply/app/src/main/res/values/type.xml

```diff
@@ -81,4 +81,11 @@
         <item name="android:textColor">@color/color_on_surface_emphasis_high</item>
     </style>
 
+    <style name="TextAppearance.Reply.EmailSubjectTitle" parent="TextAppearance.Material3.BodyLarge">
+        <item name="fontFamily">@font/work_sans</item>
+        <item name="android:textSize">18sp</item>
+        <item name="android:textColor">@android:color/black</item>
```

Comment:
instead of hardcoding colors, we pretty much always have to be using theme attributes. e.g. `?attr/colorOnSurface`, `?attr/colorOnSurfaceVariant`, etc. that way it adjusts correctly for light vs dark mode (otherwise this would probably be invisible black text on top of a black surface), and also so that it responds to dynamic color mode.